docs/application/native/guides/security/overview.md

```diff
@@ -6,26 +6,32 @@ The security features introduce how private information remains private, and how
 You can use the following security features in your native applications:
 
 - [Secure Key Management](secure-key.md)
-You can provide a secure repository for keys, certificates, and sensitive data related to users and their password-protected applications. You can also use secure cryptographic operations for non-exportable keys without revealing the key values to clients.
+
+  You can provide a secure repository for keys, certificates, and sensitive data related to users and their password-protected applications. You can also use secure cryptographic operations for non-exportable keys without revealing the key values to clients.
 
 - [Privilege Information](privilege.md)
-You can retrieve information about existing permissions granted by a privilege. The information can be delivered to the user as a notification.
+
+  You can retrieve information about existing permissions granted by a privilege. The information can be delivered to the user as a notification.
 
 - [Privacy-related Permissions](requesting-permissions.md)
```

Comment:
As requesting-permissions.md is changed into privacy-related-permissions.md, this link also should be changed.

Reply:
Done